jupyter_forward/core.py

```diff
@@ -182,7 +189,7 @@ def start(self):
 
     def _get_hostname(self):
         if self.launch_command:
-            return r'\$(hostname -f)'
+            return r'$(hostname -f)'
```

Comment:
actually, this should be a normal string now?
```suggestion
            return '$(hostname -f)'
```